pegasus/sites.v3/code.org/views/resources_tabs.haml

```diff
@@ -2,7 +2,8 @@
 -# "design-system-pegasus.scss" stylesheet.
 
 %h2=hoc_s(:resources_tabs_heading)
-%p=hoc_s(:resources_tabs_desc, markdown: :inline, locals: {sign_up_url: CDO.studio_url("/users/sign_up")})
+%p.desc
+  =hoc_s(:resources_tabs_desc, markdown: :inline, locals: {sign_up_url: CDO.studio_url("/users/sign_up")})
```

Comment:
So this part is centered as well as the h2? I thought that is different from what's in your screenshots

Reply:
Ah, I pulled it down to understand this magic! The h2 and p.desc is the top part of this screenshot
<img width="674" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/9142121/684c81a0-808a-4bb1-b677-30a7c9192765">

Reply:
Yeah! The only place (so far) where the `h2` and `p.desc` is centered is on the Teach page so I added the explicit styles on the `teach-styles.scss` stylesheet — all of the other pages are left-aligned so it's working as expected there.

Reply:
Got it, thank you!